scripts/sns/sns.coffee

```diff
@@ -86,7 +86,12 @@ class SNS
     @robot.emit 'sns:notification', message
     @robot.emit 'sns:notification:' + message.topic, message
 
+  channelID: (name) ->
+    temp = name.toLowerCase().replace(/\s/g, "_")
+    "#{process.env.HUBOT_HIPCHAT_JID.split("_")[0]}_#{temp}@conf.hipchat.com"
```

Comment:
We probably don't want hipchat specific code here. The point of this code was to make it easier for the sns message to be composed. Hipchat makes people dig to find the JID of a room.